hamocc/carchm.F90

```diff
@@ -20,7 +20,8 @@
 
       SUBROUTINE CARCHM(kpie,kpje,kpke,kbnd,                                  &
                         pdlxp,pdlyp,pddpo,prho,pglat,omask,                   &
-                        psicomo,ppao,pfu10,ptho,psao)
+                        psicomo,ppao,pfu10,ptho,psao,                         &
+                        pflxdms,pflxbromo)
 !******************************************************************************
```

Comment:
I would suggest to keep the `pflxdms` and `pflxbromo` out of this PR, since they will be included via the nuopc PR, right? At the moment, they aren't used.

Reply:
Good point. I've removed them.